Services/Mail/classes/Form/class.ilIncomingMailInputGUI.php

```diff
@@ -83,7 +76,7 @@ public function render()
     /**
      * @inheritdoc
      */
-    public function getItemByPostVar($a_post_var)
+    public function getItemByPostVar($a_post_var) : mixed
```

Comment:
The `mixed` type is not supported in PHP 7.4, which we'll have to support as well in ILIAS 8.